kpops/cli/main.py

```diff
@@ -187,6 +188,30 @@ def create_pipeline_config(
     return pipeline_config
 
 
+@app.command(
+    help="Generates json schemas. "
+    "The schemas can be used to enable support for kpops files in a text editor."
+)
+def gen_schema(
+    path: Path = typer.Argument(
```

Comment:
what's your opinion on printing the schema to stdout instead of saving it to a file? of course we could then only generate one schema at a time (for pipeline _or_ config)

Reply:
the ideal "end game" use case I have in mind: 
1. open any deployment project in my code editor
2. it automatically executes `kpops schema ... components` generating the pipeline schema including all the project-specific custom components
3. the schema is loaded into YAML language server
4. start editing with validation + autocompletion for every component

Reply:
Sounds cool. I can enforce that only 1 flag can be set at a time.